### DISCUSSION TITLE
Welcome to turkey Discussions!

### DISCUSSION THREAD

**Discussion by yanfengliu** [yanfengliu/turkey#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by steevessaillant** [comment]
Hi yanfengliu. hope you are doing well.

I see this in your code : 

cy.visit("/localDemo");
    cy.wait(waiting_time.page_ready);
    cy.matchImageSnapshot("original_state");

Why don't younuse the async pattern as explained at cypress.io

The good pratice would be:

//let the promise return in the then clause!   :)
cy.visit("/localDemo").
then(() => {
       cy.matchImageSnapshot("original_state");
});

Try it! If you wish.

Steeves Saillant
